app.py

```diff
@@ -1,20 +1,47 @@
 #!flask/bin/python
 import json
+
 from functools import wraps
 from flask import Flask, jsonify, redirect, request, render_template, url_for, abort
 import os
 from collections import OrderedDict
 os.chdir(os.path.abspath(os.path.dirname(__file__)))
 
 import ncrmodel
+import train
+from generate_qsub_job import generate_qsub_job
+
+CONST_HOMEDIR = os.environ['HOME']
+
+CONST_FASTTEXT_WORD_VECTOR_FILEPATH = "{}/opt/ncr/model_params/pmc_model_new.bin" #Relative to $HOME
+CONST_NEGFILE_FILEPATH = "{}/wikipedia_small.txt" #Relative to $HOME
+CONST_UPLOADED_OBO_DIR = "{}/uploaded_obo" #Relative to $HOME
+CONST_PARAMS_FILEPATH = "{}/trained_model_param" #Relative to $HOME
+CONST_QSUB_FILEPATH = "{}/qsub" #Relative to $HOME
 
 app = Flask(__name__)
 
-#'''
+'''
 model = ncrmodel.NCR.loadfromfile('model_params', 'model_params/pmc_model_new.bin')
 threshold = 0.6
-#'''
+'''
+
+#Stored in a form of {"object": model object, "threshold": threshold value}
+NCR_MODELS = {}
 
+#For now, to be later fixed...
```

Comment:
When later?

Reply:
I revised this comment clarifying its meaning - see line 18 in `app.py`. In this early version of the multi-model API, models are to be added by modification of the lines below defining the object `NCR_MODELS`. In upcoming Pull Requests, API endpoints will be created for model training, and after successful model training, these models will be automatically added to the `NCR_MODELS` object.